### ISSUE TITLE
Improve base typographic rhythm

### ISSUE THREAD

**Issue by iamthomasbishop** [wordpress-mobile/gutenberg-mobile#550]
When doing some investigation during some recent Headings font work, I noticed that our base typographic rhythm is a bit cramped. I compared against the following:

* **Core GB**: 16 / 1.8
* **Reader (web/mobile)**: 17 / 1.6
* **Mobile GB (current)**: 16 / 1.375
* **Mobile GB (proposed)**: 16 / 1.5

For a visual reference, set on a canvas width of 375:

<img width="1993" alt="image" src="https://user-images.githubusercontent.com/1200432/52377075-664f2780-2a29-11e9-92f9-996b7708dba6.png">

I'm tempted to defer to what Reader web/mobile has in terms of line-height, but considering the base font-size is 16 in our case, the 1.5 line-height ends up being a nice round 24pt, which aligns really well with our 16pt grid and our Block top/bottom margins (12pt).

<img width="945" alt="screen shot 2019-02-06 at 4 16 25 pm" src="https://user-images.githubusercontent.com/1200432/52377487-a531ad00-2a2a-11e9-80c2-345a30da3b21.png">

### Punch List

- [ ] Change base line-height of paragraph text to 1.5. If we have to define a static px value, let's go with 24.

Bonus points if we can set up a nice type system that scales well with Lists, Quotes, etc. when we get to those blocks.

**Comment by kjellr** [comment]
My general rule of line height: pick what seems to be just comfortable enough, and add a point. 🙂

`16 / 1.5` seems a bit too tight to me, and I agree that `1.8` might be too much here. Can we try working out a nice round number closer to `1.6`?

**Comment by iamthomasbishop** [comment]
> My general rule of line height: pick what seems to be just comfortable enough, and add a point.

That's a nice rule 😄 While I was hoping to come up "one standard to rule them all", I'm not sure that "because it aligns to our 8/24 grid" is good enough reason to stick to that pipe dream – especially considering dynamic type sizes on iOS/Android and such.

1.6 does feel pretty good, even if it's not _exactly_ on the 8pt grid. Considering this ratio is also in-line with what we're doing on Reader – and upon reviewing what Apple does for iOS "Reader Mode" (also 1.6 ratio) – I'm going to adjust the proposal to 1.6.

Thanks for the feedback/insight! I think this is also something that we should feel comfortable pushing upstream to Core. // cc @jasmussen

**Comment by jasmussen** [comment]
Created https://github.com/WordPress/gutenberg/pull/13765, let me know if I got that right!

**Comment by SergioEstevao** [comment]
@iamthomasbishop is this still a thing we want to do?

**Comment by iamthomasbishop** [comment]
@SergioEstevao Yes, I would still like to do this — it will help a lot with readability and consistency.
